packages/rancher-webhook/package.yaml

```
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.2.0/rancher-webhook-0.2.0.tgz
+url: https://github.com/rancher/webhook/releases/download/v0.2.1-rc1/rancher-webhook-0.2.1-rc1.tgz
 version: 1.0.0
```

rancher-webhook 1.0.0+up0.2.0 is already release https://github.com/rancher/charts/tree/release-v2.6/charts/rancher-webhook/rancher-webhook

can you bump this version to 1.0.1
```suggestion
version: 1.0.1
```

Also, why there is no new entry added to the index.yaml file?